modules/andes-core/broker/src/main/java/org/wso2/andes/amqp/QpidAndesBridge.java

```diff
@@ -370,10 +374,19 @@ public static void createQueue(AMQQueue queue) throws AMQException {
             log.debug("AMQP BRIDGE: create queue: " + queue.getName());
         }
         try {
-            Andes.getInstance().createQueue(AMQPUtils.createAndesQueue(queue));
+            //Validating the queue name, when creating queues from a client
+            if (Pattern.matches("[[a-zA-Z]+[^(\\x00-\\x80)]+[0-9_\\-/#*:.?&()]+]+", queue.getName())) {
+                Andes.getInstance().createQueue(AMQPUtils.createAndesQueue(queue));
+                log.info("Queue " + queue.getName() + " created successfully.");
+            } else {
+                log.error("Queue name cannot contain any of following symbols ~!@#;%^*()+={}|<>\"', and space \n");
+                throw new AMQException(AMQConstant.NOT_ALLOWED,
+                        "Queue name cannot contain any of following symbols ~!@#;%^*()+={}|<>\"', and space \n");
+
+            }
```

Comment:
Shall we move this as method to AMQUtils so that we can reuse it in QueueBindHandler?